docs/installation.md

```diff
@@ -0,0 +1,293 @@
+# Installation
+
+This document provides detailed instructions on how to install the project. Follow these steps to set up the project on your local system.
+
+## Prerequisites
+
+[Homebrew](https://brew.sh) is the simplest way to install system dependencies needed for this project.
+
+[Chocolatey](https://community.chocolatey.org/) is a good equivalent to Homebrew for package management in Windows.
+
+## System Requirements
+
+
+- **Node/NPM** (recommend using nvm)
+- **MariaDB** (using MariaDB in docker is also good)
+- **Java** (for Liquibase / OpenAPI  Generator)
+- **Liquibase**
+- **OpenAPI Generator**
+- **pyenv**
+- **wget** *(optional)*
+
+### Mac
+
+```bash
+brew install nvm
+brew install java
+brew install liquibase
+brew install pyenv
+brew install wget
+
+# skip if you wish to install via docker
+brew install mariadb@10.8
+
+```
+
+### Windows
+
+```bash
```

Comment:
Were you able to test this on Windows? I'm a little hesitant to add instructions as we only test on Mac and unix systems. Maybe to add a disclaimer like "we don't test on windows, your mileage may vary, but consider using `choco` to install these tools"

Reply:
Good idea, have added a disclaimer to that section now!

Reply:
Is there an unpushed commit, I don't see a disclaimer here.